lib/Yield/Calls/Slices.cpp

```diff
@@ -0,0 +1,133 @@
+/// \file Slices.cpp
+/// \brief
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include <unordered_map>
+
+#include "llvm/ADT/BreadthFirstIterator.h"
+#include "llvm/ADT/PostOrderIterator.h"
+
+#include "revng/Yield/Calls/Slices.h"
+
+using NodeView = const yield::Graph::Node *;
+
+static yield::Graph::Node *
+nodeHelper(yield::Graph &Graph, const yield::Graph::Node *Source) {
```

Comment:
I went with `copyNode` instead since it's closer to what the helper is doing. Thanks for pointing this out!

https://github.com/revng/revng/pull/191/commits/4de58cdb1116e6e413cc218fb994592acca49b80